Easydict/App/Localizable.xcstrings

```diff
@@ -4840,7 +4840,7 @@
         }
       }
     },
-    "request_screen_capture_access_description" : {
+    "request_screen_capture__description" : {
```

Comment:
You seem to have changed the original string.

Reply:
I don't know what to do now. I am not very familiar with Github. I just wanted to update the translation. Can you please fix it or should I make a new PR as I have no idea what to do now?

Reply:
Thanks a lot Jerry for your help!

I just edited the file here on Github.

Xcode is meant to be for developers. Xcode is really unnecessary in my opinion as it is too big and really complicated for someone who just wants to translate something. There are other lightweight and small apps, that can be used for translating apps:

[Loca Studio](https://apps.apple.com/sk/app/loca-studio/id1465684707?l=sk&mt=12) for example. I believe lot of other users who can contribute with translation including myself finds Xcode really complicated and it is much easier to just download a file for translation and import it to for example to the free Loca Studio application. Maybe you could try it and update the guide? I think a simple way of translating could make more people to help to translate Easydict to their languages.